libnd4j/include/ops/declarable/helpers/cpu/transforms.cpp

```diff
@@ -828,6 +856,9 @@ template void mirrorPad<float>(const NDArray<float>& input, const NDArray<float>
 template void mirrorPad<float16>(const NDArray<float16>& input, const NDArray<float16>& paddings, NDArray<float16>& output, const int mode);
 template void mirrorPad<double>(const NDArray<double>& input, const NDArray<double>& paddings, NDArray<double>& output, const int mode);
 
+template void concat<float>(const std::vector<NDArray<float>*>& inArrs, NDArray<float>& output, const int axis);
+template void concat<float16>(const std::vector<NDArray<float16>*>& inArrs, NDArray<float16>& output, const int axis);
+template void concat<double>(const std::vector<NDArray<double>*>& inArrs, NDArray<double>& output, const int axis);
```

Comment:
Would you consider adding template for IntBuffer and LongBuffer backed arrays?
With this work, it seems like I could simply drop #5686

Reply:
Hi @emillynge 
I am afraid it is too hard to provide such functionality on the present stage of libnd4j development, since this would require a lot off additional templates implementations - corresponding array templates, types conversions, assignment/copy operators and so on.